os-autoinst-obs-auto-submit

```diff
@@ -246,7 +246,7 @@ submit() {
             handle_auto_submit "$package" || rc=$?
         done
         # delete package from staging project
-        is_staging_specified && $osc rdelete -m "Cleaning up $package from $staging_project for next submission" "$staging_project" "$package"
+        [[ $must_cleanup_staging ]] && $osc rdelete -m "Cleaning up $package from $staging_project for next submission" "$staging_project" "$package"
```

Comment:
...or is there a reason to not just use the `is_staging_specified` function and introduce yet another variable?